test/e2e_node/device_plugin_test.go

```diff
@@ -803,13 +803,10 @@ func testDevicePluginNodeReboot(f *framework.Framework, pluginSockDir string) {
 
 		ginkgo.BeforeEach(func(ctx context.Context) {
 			ginkgo.By("Wait for node to be ready")
-			gomega.Eventually(ctx, func(ctx context.Context) error {
-				_, err := e2enode.TotalReady(ctx, f.ClientSet)
-				if err != nil {
-					return err
-				}
-				return nil
-			}, time.Minute, time.Second).Should(gomega.Equal(1), "one node should be ready")
+			gomega.Eventually(ctx, e2enode.TotalReady).
```

Comment:
`e2enode.TotalReady` returns two value. Does `gomega.BeEquivalentTo` automagically handles the `err` part?

Reply:
It only checks the first return value, I assume it’s fine to skip err, other e2e tests do the same.

Reply:
The error is checked by `gomega.Eventually`. The matcher only gets invoked when the error is nil.

/lgtm

Reply:
Good to know, thanks!